src/Containers-OrderPreservingDictionary/CTOrderPreservingIdentityDictionary.class.st

```diff
@@ -1,5 +1,21 @@
 "
-An IdentityDictionary version that uses == instead of = for key comparing.
+I am an IdentityDictionary version that uses == instead of = for key comparing. That means I consider two keys to be equal only if they are the same object.
```

Comment:
This is kind of redundant. A normal dictionary does equality comparisons (`=`). An identity dictionary by definition uses identity comparisons (`==`).

I'd rephrase it maybe like:

> I'm a dictionary that uses identity comparison (==) on my keys instead of equality.
> That means I consider two keys to be equal only if they are the same object.